resources/qml/PrinterSelector/MachineSelectorButton.qml

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.7
+import QtQuick.Controls 2.1
+
+import UM 1.1 as UM
+import Cura 1.0 as Cura
+
+Button
+{
+    id: machineSelectorButton
+
+    width: parent.width
+    height: UM.Theme.getSize("action_button").height
+    leftPadding: Math.round(1.5 * UM.Theme.getSize("default_margin").width)
```

Comment:
Soo thick margin?

Reply:
Done in 5b940b524204ca56b5fafe05bad28d5e477b8ea7

Reply:
Done in 5b940b524204ca56b5fafe05bad28d5e477b8ea7